packages/sso-menu/src/SSOMenu.tsx

```diff
@@ -0,0 +1,279 @@
+import { uiColors } from '@leafygreen-ui/palette';
+import { css, cx } from '@leafygreen-ui/emotion';
+
+const iconRef = createDataProp('icon-ref');
+
+const buttonReset = css`
+  appearance: none;
+  background: none;
+  border: 0px;
+  position: relative;
+  padding: 0px;
+
+  &:hover:before {
+    transform: scale(1);
+  }
+
+  &:active {
+    color: ${uiColors.gray.dark2};
+
+    &:before {
+      transform: scale(1);
+    }
+
+    & ${iconRef.selector} {
+      color: ${uiColors.gray.dark1};
+    }
+  }
+
+  &:focus {
+    outline: none;
+
+    &:before {
+      background-color: #63b0d0;
+      transform: scale(1);
+    }
+  }
+
+  &::-moz-focus-inner {
+    border: 0;
+  }
+
+  &:before {
+    content: '';
+    position: absolute;
+    top: -2px;
+    bottom: -2px;
+    left: -2px;
+    right: -2px;
+    border-radius: 50px;
+    transform: scale(0.9, 0.8);
+    transition: transform 150ms ease-in-out;
+    background-color: ${uiColors.gray.light2};
+  }
+`;
+
+const menuButtonStyle = css`
+  height: 29px;
+  padding-left: 14px;
+  padding-right: 14px;
+  border: 1px solid ${uiColors.gray.light1};
+  background-color: ${uiColors.white};
+  border-radius: 14.5px;
+  cursor: pointer;
+  transition: background 150ms ease-in-out;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: ${uiColors.gray.dark2};
+  font-size: 12px;
+  position: relative;
+
+  &:focus {
+    outline: none;
+  }
+`;
+
+const menuNameStyle = css`
+  margin-right: 2px;
+  margin-left: 2px;
+`;
+
+const activeMenuButtonStyle = css`
+  background-color: ${uiColors.gray.light2};
+`;
+
+const nameStyle = css`
+  font-size: 16px;
+  color: ${uiColors.gray.dark2};
+  font-weight: bold;
+  margin: 4px 0px 2px;
+`;
+
+const truncate = css`
+  width: 162px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+const closedIconStyle = css`
+  transform: rotate(180deg);
+  transition: color 200ms ease-in-out;
+  color: ${uiColors.gray.base};
+`;
+
+const openIconStyle = css`
+  margin-top: 2px;
+  color: ${uiColors.gray.base};
+`;
+
+const accountMenuGroupStyle = css`
+  padding: 20px 20px 14px;
+`;
+
+const accountButtonStyle = css`
+  margin-top: 12px;
+  width: 100%;
+  display: inline-flex;
+  justify-content: center;
+`;
+
+const descriptionStyle = css`
+  margin: 0px;
+  font-size: 12px;
+  color: ${uiColors.gray.dark1};
+  text-decoration: none;
+`;
+
+const logoutContainerHeight = css`
+  height: 46px;
+`;
+
+const menuItems = [
+  {
+    displayName: 'Atlas',
+    description: 'cloud.mongodb.com',
+    href: 'https://cloud.mongodb.com',
+    slug: 'atlas',
+  },
+  {
+    displayName: 'University',
+    description: 'university.mongodb.com',
+    href: 'https://university.mongodb.com',
+    slug: 'university',
+  },
+  {
+    displayName: 'Cloud Support',
+    description: 'support.mongodb.com',
+    href: 'https://support.mongodb.com',
+    slug: 'support',
+  },
+] as const;
+
+export const Products = {
```

Comment:
For these types of enums this should be singular. As it's not a collection of Products but an enumeration of our Product names.

i.e. Product.Atlas is the fully qualified name as opposed to Products.Atlas is an odd mix of a collection and a name.